### DISCUSSION TITLE
Return static default value when localization key does not exist

### DISCUSSION THREAD

**Discussion by pchaudha** [shoelace-style/localize#23] [Q&A] [answered]
Is there any way to return static value provided if key does not exist?

**Comment by claviska** [comment, marked as answer]
If a term doesn't exist in the target translation or the fallback, the key will be output. There's currently no way to change this, but there's an undocumented method to check if a term exists that may help you.

https://github.com/shoelace-style/localize/blob/6e9dfa53961be985dd340d436ec0abf59d49c3e3/src/index.ts#L131-L149

I would consider this method public and stable.
